phpunit.xml.dist

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+	PHPUnit configuration for SilverStripe
+	Requires PHPUnit 5+
+	Usage:
+	 - "vendor/bin/phpunit": Runs all tests in all folders
+	 - "vendor/bin/phpunit vendor/silverstripe/framework/tests/": Run all tests of the framework module
+	 - "vendor/bin/phpunit vendor/silverstripe/framework/tests/filesystem": Run all filesystem tests within the framework module
+	 - "vendor/bin/phpunit vendor/silverstripe/framework/tests/filesystem/FolderTest.php": Run a single test
+	 - "vendor/bin/phpunit --coverage-html assets/": Generate coverage report (replace <dash> with "-", requires xdebug)
+	More information:
+	- http://www.phpunit.de/manual/current/en/textui.html
+	- http://doc.silverstripe.org/framework/en/topics/testing/#configuration
+	It is safe to remove this file for normal website operation.
+-->
+<phpunit bootstrap="vendor/silverstripe/framework/tests/bootstrap.php" colors="true">
+
+	<testsuite name="Default">
+		<directory>vendor/silverstripe/blog/tests</directory>
+		<directory>vendor/silverstripe/widgets/tests</directory>
+		<directory>vendor/silverstripe/content-widget/tests</directory>
+		<directory>vendor/silverstripe/spamprotection/tests</directory>
+		<directory>vendor/silverstripe/akismet/tests</directory>
+		<directory>vendor/silverstripe/comments/tests</directory>
+		<directory>vendor/silverstripe/comment-notifications/tests</directory>
+		<directory>vendor/colymba/gridfield-bulk-editing-tools/tests</directory>
+	</testsuite>
+
+    <!-- silverstripe core components -->
+    <testsuite name="core">
+		<directory>vendor/silverstripe/framework/tests/php/</directory>
+		<directory>vendor/silverstripe/config/tests</directory>
+		<directory>vendor/silverstripe/assets/tests/php/</directory>
+        <directory>vendor/silverstripe/versioned/tests/php/</directory>
+    </testsuite>
+
+    <!-- silverstripe admin components -->
+    <testsuite name="admin">
+        <directory>vendor/silverstripe/cms/tests/</directory>
+        <directory>vendor/silverstripe/admin/tests/php/</directory>
+        <directory>vendor/silverstripe/campaign-admin/tests/php/</directory>
+        <directory>vendor/silverstripe/asset-admin/tests/php/</directory>
+        <directory>vendor/silverstripe/graphql/tests/</directory>
+        <directory>vendor/silverstripe/siteconfig/tests/php/</directory>
+        <directory>vendor/silverstripe/reports/tests/</directory>
+    </testsuite>
+
+	<groups>
+		<exclude>
+			<group>sanitychecks</group>
+		</exclude>
+	</groups>
```

Comment:
I think we can just include a "recipe-blog" testsuite for the modules we're including and ignore the core ones. You can also remove the `<groups>` and the comments at the top of the file I reckon